c2corg_ui/static/js/style/autoheight.js

```diff
@@ -0,0 +1,66 @@
+goog.provide('app.autoHeightDirective');
+goog.provide('app.AutoHeightController');
+
+goog.require('app');
+
+
+/**
+ * Apply automatic height depending on text to all textarea tags
```

Comment:
There is no explicit length limitation for the summary attribute. But since a good summary should remain short, the default textarea size and behaviour look enough to me.

>  The simple fact that a textarea will open according to the text is very handy imho. Have you tested it?

Not yet but I can admit it can be quite handy (no clue for mobile usage though).

I am not that sure it is really relevant that the textarea height grows with no limit: for a very long piece of text it does not make much difference to scroll inside the textarea than in the whole page. In addition if the textarea gets really high it will become a pain in the neck to reach the formatting toolbar (placed at the top of the textarea?) that we will have to restore one day or another to help users  format the content (see https://github.com/c2corg/v6_ui/issues/305).
Or perhaps this toolbar will always be visible, floating at the top of the visible part of the current textarea?

I wonder if setting a max-height for those textareas is not needed anyway.